CDC-Data-Reconciliation-Backend/compare.py

```diff
@@ -122,11 +124,29 @@ def comp(state_dict, cdc_dict):
                 # If a case has different attributes between state and CDC DBs, mark it as such
                 if state_attribute != cdc_attribute:
                     results.append(CaseResult(state_case_id, state_row['EventCode'], state_row['EventName'], state_row[
-                                   'MMWRYear'], state_row['MMWRWeek'], "Case has different attributes between State and CDC CSV Files", "3"))
+                                   'MMWRYear'], state_row['MMWRWeek'], "Case differs on _ between State and CDC CSV Files", "3"))
                     
                     # making sure to also count this discrepancy in the stats.csv file
                     stats[state_row['EventCode']]['totalWrongAttributes'] += 1
+                    wrong_att = True
                     break
+                
+            if (wrong_att):
+                # Skipping the first attribute
+                att_list = []
+                for attribute in state_row[1:]:
+                    state_attribute = state_row[attribute]
+                    cdc_attribute = cdc_dict[state_case_id][attribute]
+                    
+                    # Putting non-matching attributes into a list
+                    if state_attribute != cdc_attribute:
+                        att_list.append(attribute)
```

Comment:
I would not do this logic outside of the for loop on line 109. Waffy just made changes to this code in his PR as well which would not work well with this code. Instead of having wrong_att as a boolean, make it an empty array on line 107. Then add to this array whenever the if statement on line 125 is satisfied. Move the logic inside of this if statement (Lines 126-130) to be below the if statement on line 134. Also make sure the break statement on line 132 is removed so that wrong_att will collect all of the attributes that are wrong and not just one.